docs/source/user_guide/installation/index.rst

```diff
@@ -50,8 +50,165 @@ Following GNU/Linux and macOS solvers are optionally available:
+
+.. _install.examples_dependencies:
+
+Examples dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[examples]"``
+
+================================================================================ ================== ================== ===================================================================================================================================================================================
+Dependency                                                                       Minimum Version    pip extra          Notes
+================================================================================ ================== ================== ===================================================================================================================================================================================
+`pre-commit <https://pre-commit.com/index.html>`__                               -                  examples           For managing and maintaining multi-language pre-commit hooks.
+`black <https://black.readthedocs.io/en/stable/>`__                              -                  examples           For code formatting.
+`jupyter <https://docs.jupyter.org/en/latest/>`__                                -                  examples           For example notebooks rendering.
+================================================================================ ================== ================== ===================================================================================================================================================================================
+
+.. _install.dev_dependencies:
+
+Dev dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[dev]"``
+
+================================================================================ ================== ================== ===================================================================================================================================================================================
+Dependency                                                                       Minimum Version    pip extra          Notes
+================================================================================ ================== ================== ===================================================================================================================================================================================
+`numpy <https://numpy.org/>`__                                                   1.16.0             dev                For complex computation handling.
+`scipy <https://docs.scipy.org/doc/scipy>`__                                     2.8.2              dev                Provides algorithms for optimization, integration, interpolation etc.
+`pandas <https://pandas.pydata.org/docs/>`__                                     0.24.0             dev                For data manipulation & analysis.
```

Comment:
these are not dev dependencies since they are always installed anyway

Reply:
Thanks @tinosulzer got it, removed them from `[all]` as well.